ambari-web/app/controllers/main/host/details.js

```diff
@@ -1775,6 +1775,16 @@ App.MainHostDetailsController = Em.Controller.extend(App.SupportClientConfigsDow
         groups.push(group);
       }
     });
+    if (App.get('isHaEnabled') && installedServiceNames.contains('HDFS')) {
```

Comment:
core-site type is added in constructZookeeperConfigUrlParams(), but problem is that zooKeeperRelatedServices missing HDFS service. Since the HDFS configs are dynamical I would modify constructZookeeperConfigUrlParams method:
const zooKeeperRelatedServices = this.get('zooKeeperRelatedServices').slice(0);
if (App.get('isHaEnabled')) {
      zooKeeperRelatedServices.push({
          serviceName: 'HDFS',
         typesToLoad: ['core-site'],
         typesToSave: ['core-site']
    })
    }
zooKeeperRelatedServices.forEach(function (service) {...

Reply:
Hi @atkach ,
I have corrected the file as per the review comment.
Can you please review the latest code change.

Reply:
Also, you need to copy the array (*.slice(0)) and not mutate the original.

Reply:
Hi @atkach  ,
I have implemented the changes you have suggested.